physics/GFS_rrtmgp_thompsonmp_pre.F90

```diff
@@ -0,0 +1,297 @@
+! ########################################################################################
```

Comment:
@ChunxiZhang-NOAA 
GFS_rrtmgp_thompsonmp.F90 has been replaced with GFS_rrtmgp_cloud_mp.F90.

It looks like the new functionality added alongside "mraerosol" was added to both  GFS_rrtmgp_cloud_mp.F90 and GFS_rrtmgp_thompsonmp.F90. Since GFS_rrtmgp_thompsonmp.meta wasn't modified, I'm assume that you're not using this scheme and we can remove it from here?

Reply:
I don't think GFS_rrtmgp_thompsonmp_pre.F90 is used. I will delete this file.